core/frontend/src/components/video-manager/VideoStreamCreationDialog.vue

```diff
@@ -281,14 +300,35 @@ export default Vue.extend({
     addNewEndpoint() {
       this.validateForm()
       this.stream_endpoints.push('')
+      this.set_default_address_for_stream(this.stream_endpoints.length - 1, StreamType.UDP)
+      this.validateForm()
     },
     removeEndpoint(index: number) {
       this.validateForm()
       this.stream_endpoints.splice(index, 1)
+      this.validateForm()
     },
     showDialog(state: boolean) {
       this.$emit('visibilityChange', state)
     },
+    set_default_address_for_stream(index: number, stream_type: StreamType) {
+      switch (stream_type) {
+        case StreamType.UDP:
+          if (!this.stream_endpoints[index].includes('udp://')) {
+            // Vue.set() forces the update of a nested property
+            Vue.set(this.stream_endpoints, index, `udp://${this.user_ip_address}:5600`)
+          }
+          break
+        case StreamType.RTSP:
+          if (!this.stream_endpoints[index].includes('rtsp://')) {
+            // Vue.set() forces the update of a nested property
+            Vue.set(this.stream_endpoints, index, 'rtsp://0.0.0.0:8554/video0')
```

Comment:
Instead of `0.0.0.0`, since [mavlink-camera-manager t3.7.0](https://github.com/bluerobotics/mavlink-camera-manager/releases/tag/t3.7.0) it should be the adapter's IP (e.g. `192.168.2.2` by default for the static ethernet IP)